saphir/src/redirect.rs

```diff
@@ -106,6 +108,32 @@ impl Builder {
         self
     }
 
+    #[inline]
+    pub fn cookie(mut self, cookie: Cookie<'static>) -> Builder {
+        if self.cookies.is_none() {
+            self.cookies = Some(CookieJar::new());
+        }
+
+        self.cookies.as_mut().expect("Should not happens").add(cookie);
+
+        self
```

Comment:
Could be simplified like this:
```suggestion
        self.cookies_mut().add(cookie);
        self
```